ethereum/dex/trades.sql

```diff
@@ -655,6 +655,27 @@ WITH rows AS (
 
         UNION ALL
 
+        -- 1inch LP
+        SELECT
+            evt_block_time,
+            '1inch LP' AS project,
+            '1' AS version,
+            'DEX' AS category,
+            sender AS trader_a,
+            NULL::bytea AS trader_b,
+            result AS token_a_amount_raw,
+            amount AS token_b_amount_raw,
+            NULL::numeric AS usd_amount,
+            dstToken AS token_a_address,
```

Comment:
camelCase variable names must be in quotes

Reply:
Thx, fixed!

Reply:
Also, one of "dstToken" and "srcToken" seems to be always `0`. Why is that? Maybe you need to preprocess the events somehow to conform them to this table?